packages/orbit-components/src/Tabs/components/TabList/index.tsx

```diff
@@ -15,6 +15,8 @@ const TabList = ({
   margin,
   dataTest,
   fullWidth,
+  ariaLabel = "Tabs",
```

Comment:
Do we want to keep the default value? 🤔 What value does it bring?

Reply:
True, I guess it's not really useful to have there "Tabs", I'll remove the default value.

We should

Reply:
It's probably a Breaking change, but 🤷‍♂️ I guess it does not harm

Reply:
Value removed, commit and migration guide updated.